src/extent.c

```diff
@@ -1105,6 +1126,13 @@ extent_grow_retained(tsdn_t *tsdn, arena_t *arena,
 		goto label_err;
 	}
 
+	/*
+	 * There are situations in which the Linux kernel will include
```

Comment:
Really, I guess just losing "there are situations", since the kernel always does it.